src/main/java/de/hsrm/vegetables/my_food_coop_service/repositories/OffsetLimit.java

```diff
@@ -0,0 +1,74 @@
+package de.hsrm.vegetables.my_food_coop_service.repositories;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+
+public class OffsetLimit implements Pageable {
```

Comment:
I mean, it's cool that you've implemented a whole class just to use offset and limit from the users request, but... couldn't you have just used the default interface with pageSize and pageNumber and for pageNumber do something like `Math.floor(offset  / limit)` or something? That sounds like a way easier approach in my books

Reply:
Done